lib/ansible/modules/files/synchronize.py

```diff
@@ -185,7 +185,9 @@
      encounters an error. Those synchronizing large numbers of files that are willing to trade safety for performance should call rsync directly.
    - link_destination is subject to the same limitations as the underlaying rsync daemon. Hard links are only preserved if the relative subtrees
      of the source and destination are the same. Attempts to hardlink into a directory that is a subdirectory of the source will be prevented.
-
+seealso:
+- module: copy
+- module: win_robocopy
```

Comment:
Not sure if we want to link `win_robocopy`. Technically it does do synchronization but that's only on remote to remote not local to remote.

Reply:
So the **synchronize** module was mentioned in the **win_robocopy** module docs, and I tried to ensure we have a reverse link as well. I don't mind removing it though.

Reply:
I don't mind either way, as long as it's clear win_robocopy won't work for local to remote synchronization.